### ISSUE TITLE
The structure of the wizard is not proper, aria-labelledby is referencing the id of one of the tab content header, instead of the wizard header

### ISSUE THREAD

**Issue by aognianova** [vmware/clarity#4683]
## Describe the bug

This is an issue with accessibility. We got a report from AccessLevel where they note that the structure of the wizard is not proper. **aria-labelledby** is referencing the id of one of the tab content header, instead of the modal header for better context.
Because the dialogs title is often visually apparent but may be difficult to locate using assistive technology such as screen readers, a title must be provided through a proper accessible manner to ensure that the purpose of the modal dialog is understood.
Ensure that proper heading is referenced via aria-labelledby from the element representing the dialog.

## How to reproduce

1.  Go to [https://clarity.design/documentation/wizards](url)
2.  Scroll down to 'Code & Examples'
3.  Click on 'X-Large Wizard'
4.  Inspect the wizard
5.  See that the dialog element has aria-labelledby attribute that is referencing the id of the tab content header, instead of the wizard title.

![WizardTitle_LabelledBy](https://user-images.githubusercontent.com/65155173/83029762-b6e1fe00-a03b-11ea-8393-9d169e29b758.png)

## Expected behavior

```
<div _ngcontent-clarity-c167="" role="dialog" aria-modal="true" class="modal-dialog ng-tns-c167-6 ng-trigger ng-trigger-fadeDown modal-xl" aria-hidden="false" aria-labelledby="wizard-title-id">

<h1 id="wizard-title-id" class="clr-wizard-title"><clr-wizard-title>XL Wizard (Default)</clr-wizard-title></h1>

```
## Versions

**App**

- Angular: [9]
- Node: [10.16.1]
- Clarity: [3.0.0]

**Device:**

- Type: [PC]
- OS: [Windows 10]
- Browser [chrome]
- Version [81]

**Comment by gnomeontherun** [comment]
Hi is this a VMware product report? If so, this was done purposefully and should probably not be considered an issue, but it would be best work with the accessibility contact to resolve this report.

**Comment by aognianova** [comment]
> Hi is this a VMware product report? If so, this was done purposefully and should probably not be considered an issue, but it would be best work with the accessibility contact to resolve this report.

Hi!
The report is from an external vendor (Level Access). The VPAT severity of the issue is 8, that is A-level defect and is blocker to people with disabilities. It is not an option to resolve it as 'not an issue'.

**Comment by gnomeontherun** [comment]
Hi, I understand Level Access did the testing, but was the testing done for a VMware product? If so, we have a process to handle these reports that should be handled through the accessibility team for traceability and verification. It is difficult to know for sure over GitHub, which is why I asked.

This is an example of a functionality that was implemented under the guidance of the accessibility team, which means it challenges their previous recommendation. If this not a VMware product, then I'll have to review it with our accessibility team.

**Comment by aognianova** [comment]
Yes, the testing was done for a VMware product - Site Recovery Manager 8.3

**Comment by chlane** [comment]
The Wizard dialog actually has 2 titles. Fortunately, aria-labelledby can accept multiple values e.g. aria-labelledby="title1 title2". I recommend referencing both titles with aria-labelledby. See https://www.w3.org/TR/wai-aria-1.1/#aria-labelledby.

**Comment by gnomeontherun** [comment]
Clarity cares deeply about accessibility, and we work with our accessibility team to identify and triage issues. We have an internal tracking system to identify and verify accessibility issues, which has captured this already. Some issues with Clarity Angular are difficult to solve without causing side effects or breaking changes, and we have to balance concerns with fixing issues and breaking changes. As we build out Clarity Core, we are able to address these root cause accessibility issues and recommend adopting the Clarity Core implementations as they become available. We are closing this issue as we have the issues tracked internally. Please follow our development and releases to see when we release relevant Clarity Core components to make this possible.
